test/unit/shim/webframework-shim.test.js

```diff
@@ -601,7 +601,12 @@ test('WebFrameworkShim', function (t) {
         t.ok(segment.attributes)
         t.same(segment.getAttributes(), {
           'request.parameters.route.foo': 'bar',
-          'request.parameters.route.biz': 'bang'
+          'request.parameters.route.biz': 'bang',
+          'code.filepath':
+            '/Users/bcanzanella/code/node-newrelic/test/unit/shim/webframework-shim.test.js',
```

Comment:
@bizob2828 any suggestion for how to make this generic? checking the entire object won't work unless the value for code.filepath is modified. could check the length of keys and check each key/value pair with the code.filepath doing an endsWith check? thoughts?

Reply:
instead of `t.same` you could just assert the individual attrs.

```js
const attrs = segment.getAttributes()
t.equal(attrs['request.parameters.route.foo'], 'bar')
t.equal(attrs['request.parameters.route.biz'], 'bang')
```

I'm ok with that. we have other places asserting CLM data